src/plugins/repetierplugin.cpp

```diff
@@ -1,19 +1,52 @@
 #include "repetierplugin.h"
+#include <QLoggingCategory>
 #include <QString>
 
+QString RepetierPlugin::_ok = QStringLiteral("ok");
+QString RepetierPlugin::_wait = QStringLiteral("wait");
+QString RepetierPlugin::_extruderTemp = QStringLiteral("T:");
+QString RepetierPlugin::_bedTemp = QStringLiteral("B:");
+
+Q_LOGGING_CATEGORY(REPETIER_PLUGIN, "org.kde.atelier.core.firmware.repetier");
+
 QString RepetierPlugin::name() const
 {
     return QStringLiteral("Repetier");
 }
 
 RepetierPlugin::RepetierPlugin(QObject *parent)
 {
+    qCDebug(REPETIER_PLUGIN) << "Starting.. " << name();
```

Comment:
this just sounds weird coming from within the plugin like this maybe change the message to 
qCDebug(REPETIER_PLUGIN) << name() << " plugin loaded" 
 or something similar to that.